ddp-workspace/projects/ddp-dsm-ui/src/app/clinical-page/clinical-page.component.html

```diff
@@ -38,9 +38,8 @@ <h1> Clinical Orders </h1>
       <td>{{getDateFormatted(order.orderDate)}}</td>
       <td>{{getOrderStatus(order.orderStatus)}}</td>
       <td>{{order.statusDetail}}</td>
-      <td>{{order.orderMessage}}</td>
-      <td>{{order.statusMessage}}</td>
-
+      <td class="jsonEl">Click on 'download list' to view order message</td>
```

Comment:
Message can be limited to Click on 'download list' if we want keep it to one line rather than two lines.

Reply:
Wondered if these json elements heading should be more of like "Request Message" and "Response Message" rather than "Order Message" and "Status Message" ! DSM DB has it as "OrderMessage" and "StatusMessage"

Reply:
How about moving the text out of the table, maybe just above the table as a little block of text?

Reply:
I like that. 
Does this look ok ?

<img width="1760" alt="Screenshot 2024-07-01 at 11 07 16 AM" src="https://github.com/broadinstitute/ddp-angular/assets/26553609/c33c5cc3-a805-41d3-b6a3-519cbbacc745">

Reply:
Looks great!